src/main/java/frc/robot/util/BustedDigitalInput.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.util;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Timer;
+
+/**
+ * An extension of the {@code DigitalInput} class that checks for a continuous state over a specified period of time.
+ */
+public class BustedDigitalInput extends DigitalInput {
+
+    public static final double TIMEOUT = 0.019;
+
+    protected double firstOn = 0;
+
+    public BustedDigitalInput(int channel) {
+        super(channel);
+    }
+
+    @Override
+    public boolean get() {
+        if (firstOn != 0 && super.get() && firstOn + TIMEOUT < Timer.getFPGATimestamp())
+            return true;
+        if (!super.get())
+            firstOn = 0;
+        else {
+            if (firstOn == 0)
+                firstOn = Timer.getFPGATimestamp();
+        }
+        return false;
+    }
+}
```

Comment:
![where-is-it-melody-pedras](https://github.com/Spikes-2212-Programming-Guild/ChargedUp2023/assets/71877371/c508727d-4845-4e2a-8efe-38ed3ac46555)

Reply:
?

Reply:
An empty line is missing

Reply:
oh now i see
no fucking way